codec/src/main/java/io/netty/handler/codec/compression/ZlibCodecFactory.java

```diff
@@ -33,7 +33,7 @@
     private static final boolean noJdkZlibEncoder;
 
     static {
-        noJdkZlibDecoder = SystemPropertyUtil.getBoolean("io.netty.noJdkZlibDecoder", true);
+        noJdkZlibDecoder = SystemPropertyUtil.getBoolean("io.netty.noJdkZlibDecoder", false);
```

Comment:
Do you mean when `java.util.zip` was introduced? I don't remember, but I suspect it's very old as there isn't a `@since` javadoc tag there.